README.md

```diff
@@ -97,7 +97,10 @@ conda activate muse_app
 
 git clone https://github.com/Lightning-AI/stable-diffusion-deploy.git
 cd stable-diffusion-deploy
-pip install .
+
+## [Install PyTorch](https://pytorch.org) that suits your system configuration (CUDA version preferable)
+pip install torch --extra-index-url https://download.pytorch.org/whl/cu116
```

Comment:
How do you know that user is using CUDA 1.16?

Reply:
Anyway, why are you pip install torch on conda?

Reply:
conda is optional.. just for env.. one might not be using conda.. 
pip works in all kinds of env